### ISSUE TITLE
addrConn.resetTransport failed to create client transport

### ISSUE THREAD

**Issue by dongdongmeow** [coreos/etcd#8317]
so much error log  in etcd v3.0.14
grpc: addrConn.resetTransport failed to create client transport: connection error: desc = "transport: dial tcp [::]:2379: connect: network is unreachable"; Reconnecting to {"[::]:2379" <nil>}

------------------------------------
``` 
start to snapshot (applied: 1299778616, lastsnap: 1299768615)
saved snapshot at index 1299778616
compacted raft log at 1299773616
grpc: addrConn.resetTransport failed to create client transport: connection error: desc = "transport: dial tcp [::]:2379: connect: network is unreachable"; Reconnecting to {"[::]:2379" <nil>}
purged file /data/etcd.data/etcd03/member/snap/0000000000000096-000000004d7848e3.snap successfully
grpc: addrConn.resetTransport failed to create client transport: connection error: desc = "transport: dial tcp [::]:2379: connect: network is unreachable"; Reconnecting to {"[::]:2379" <nil>}
grpc: addrConn.resetTransport failed to create client transport: connection error: desc = "transport: dial tcp [::]:2379: connect: network is unreachable"; Reconnecting to {"[::]:2379" <nil>}
grpc: addrConn.resetTransport failed to create client transport: connection error: desc = "transport: dial tcp [::]:2379: connect: network is unreachable"; Reconnecting to {"[::]:2379" <nil>}
start to snapshot (applied: 1299788617, lastsnap: 1299778616)
saved snapshot at index 1299788617
compacted raft log at 1299783617
purged file /data/etcd.data/etcd03/member/snap/0000000000000096-000000004d786ff4.snap successfully
segmented wal file /data/etcd.data/etcd03/member/wal/000000000000e8f6-000000004d793cc2.wal is created
grpc: addrConn.resetTransport failed to create client transport: connection error: desc = "transport: dial tcp [::]:2379: connect: network is unreachable"; Reconnecting to {"[::]:2379" <nil>}
apply entries took too long [12.417909ms for 1 entries]
avoid queries with large range/delete range!
purged file /data/etcd.data/etcd03/member/wal/000000000000e8f1-000000004d777f10.wal successfully
grpc: addrConn.resetTransport failed to create client transport: connection error: desc = "transport: dial tcp [::]:2379: connect: network is unreachable"; Reconnecting to {"[::]:2379" <nil>}
grpc: addrConn.resetTransport failed to create client transport: connection error: desc = "transport: dial tcp [::]:2379: connect: network is unreachable"; Reconnecting to {"[::]:2379" <nil>}
start to snapshot (applied: 1299798618, lastsnap: 1299788617)
saved snapshot at index 1299798618
compacted raft log at 1299793618
purged file /data/etcd.data/etcd03/member/snap/0000000000000096-000000004d789705.snap successfully
grpc: addrConn.resetTransport failed to create client transport: connection error: desc = "transport: dial tcp [::]:2379: connect: network is unreachable"; Reconnecting to {"[::]:2379" <nil>}
grpc: addrConn.resetTransport failed to create client transport: connection error: desc = "transport: dial tcp [::]:2379: connect: network is unreachable"; Reconnecting to {"[::]:2379" <nil>}
apply entries took too long [39.835247ms for 1 entries]
avoid queries with large range/delete range!
start to snapshot (applied: 1299808619, lastsnap: 1299798618)
saved snapshot at index 1299808619
compacted raft log at 1299803619
purged file /data/etcd.data/etcd03/member/snap/0000000000000096-000000004d78be16.snap successfully
grpc: addrConn.resetTransport failed to create client transport: connection error: desc = "transport: dial tcp [::]:2379: connect: network is unreachable"; Reconnecting to {"[::]:2379" <nil>}
segmented wal file /data/etcd.data/etcd03/member/wal/000000000000e8f7-000000004d7995c1.wal is created
grpc: addrConn.resetTransport failed to create client transport: connection error: desc = "transport: dial tcp [::]:2379: connect: network is unreachable"; Reconnecting to {"[::]:2379" <nil>}
grpc: addrConn.resetTransport failed to create client transport: connection error: desc = "transport: dial tcp [::]:2379: connect: network is unreachable"; Reconnecting to {"[::]:2379" <nil>}
purged file /data/etcd.data/etcd03/member/wal/000000000000e8f2-000000004d77d837.wal successfully

```

**Comment by heyitsanthony** [comment]
@dongdongmeow What flags are passed to etcd? Where are the timestamps in the log?

It's possible this may be fixed by #8221 / 63350f5ac10a8035e9284c8385e75d2e27d669c3 / 3.2.2 or by enabling ipv6 on the host.

**Comment by heyitsanthony** [comment]
Closing due to inactivity and seems to be fixed. Please reopen if problem persists. Thanks!
